src/Core/Helpers/ChecksumCalculator.cs

```diff
@@ -26,10 +26,15 @@
 
 namespace SonarLint.VisualStudio.Core.Helpers
 {
+    public interface IChecksumCalculator
+    {
+        string Calculate(string text);
+    }
+
     /// <summary>
     /// Calculates the checksum for an issue using the same method as the SonarQube server
     /// </summary>
-    public static class ChecksumCalculator
+    public class ChecksumCalculator : IChecksumCalculator
     {
         public static string Calculate(string text)
```

Comment:
Does this still need to be `public static`?

Reply:
Yep, it's still used statically by IssueToIssueMarkerConverter and LiveIssue